styles/styles.css

```diff
@@ -1,28 +1,129 @@
 .nfd-container__block.newfold-link-prefetch .nfd-toggle-field.nfd-mb-6 {
 	display: flex;
 	flex-direction: row;
+  }
+.nfd-performance-jetpack-boost-upsell,
+.nfd-performance-jetpack-boost-container-options{
+	position: relative;
+}
+
+.nfd-root .nfd-modal{
+	z-index: 12;
+}
+
+.nfd-performance-jetpack-boost-critical-css-modal .nfd-modal__panel{
+	padding-right: 40px;
+}
+
+.nfd-performance-jetpack-boost-action-buttons-container .nfd-button--primary{
+	margin-right: 10px;
+}
+
+.nfd-performance-jetpack-boost-critical-css-modal p{
+	margin-bottom: 10px;
+}
+
+.nfd-root .nfd-modal__overlay {
+    z-index: 300;
+}
+
+.nfd-performance-jetpack-boost-critical-css-modal .nfd-modal__panel {
+    z-index: 400;
 }
 
-.nfd-performance-jetpack-boost-upsell {
+.nfd-performance-jetpack-boost-container-options .automatic-critical-css{
 	position: relative;
 }
 
-.nfd-performance-jetpack-boost-upsell::before {
-	content: "";
+.nfd-performance-jetpack-boost-container-options .automatic-critical-css:after {
+    content: '';
+    display: block;
+    width: 100%;
+    height: 100%;
+    background-color: #fff;
+    position: absolute;
+    top: 0;
+    left: 0;
+    opacity: 0.4;
+    z-index: 10;
+}
+
+.nfd-performance-jetpack-boost-container-options .automatic-critical-css .nfd-feature-upsell .nfd-button--upsell{
+	top: 0;
+    right: 0;
+    position: absolute;
+	z-index: 11;
+}
```

Comment:
We can run `wp-scripts lint-style styles/styles.css --fix` to automatically fix any formatting issues.